docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -476,77 +476,77 @@ Starting with .NET Framework 4.7.1, improvements in high contrast have been made
+  ![Screenshot of high contrast radio button before improvements.](./media/whats-new-in-accessibility/high-contrast-radio-button-before.png)
 
   After: 
 
-  ![High contrast radio button with focus after accessibility improvements](./media/radio-button-after.png)
+  ![Screenshot of high contrast radio button after improvements.](./media/whats-new-in-accessibility/high-contrast-radio-button-after.png)
 
 - <xref:System.Windows.Controls.ComboBox> control
 
   Starting with .NET Framework 4.7.1, the border of a disabled <xref:System.Windows.Controls.ComboBox> control is the same color as disabled text. For example:
 
   Before: 
 
-  ![ComboBox disabled border and text before accessibility improvements](./media/combo-disabled-before.png)
+  ![Screenshot of ComboBox disabled before accessibility improvements.](./media/whats-new-in-accessibility/combo-disabled-before.png)
 
   After:   
 
-  ![ComboBox disabled border and text after accessibility improvements](./media/combo-disabled-after.png)
+  ![Screenshot of ComboBox disabled after accessibility improvements.](./media/whats-new-in-accessibility/combo-disabled-after.png)
 
   In addition, disabled and focused buttons use the correct theme color.
 
   Before:
 
-  ![Button theme colors before accessibility improvements](./media/button-themes-before.png) 
+  ![Screenshot of button theme colors before accessibility improvements.](./media/whats-new-in-accessibility/button-theme-colors-before.png) 
 
   After: 
 
-  ![Button theme colors after accessibility improvements](./media/button-themes-after.png) 
+  ![Screenshot of button theme colors after accessibility improvements.](./media/whats-new-in-accessibility/button-theme-colors-after.png) 
 
   Finally, in .NET Framework 4.7 and earlier versions, setting a <xref:System.Windows.Controls.ComboBox> control’s style to `Toolbar.ComboBoxStyleKey` caused the drop-down arrow to be invisible. This issue is fixed starting with .NET Framework 4.7.1. For example:
 
   Before: 
 
-  ![Toolbar.ComboBoxStyleKey before accessibility improvements](./media/comboboxstylekey-before.png) 
+  ![Screenshot of Toolbar.ComboBoxStyleKey before improvements.](./media/whats-new-in-accessibility/combo-box-style-key-before.png) 
 
   After: 
 
-  ![Toolbar.ComboBoxStyleKey after accessibility improvements](./media/comboboxstylekey-after.png) 
+  ![Screenshot of Toolbar.ComboBoxStyleKey after improvements.](./media/whats-new-in-accessibility/combo-box-style-key-after.png) 
 
 - <xref:System.Windows.Controls.DataGrid> control
 
   Starting with .NET Framework 4.7.1, the sort indicator arrow in <xref:System.Windows.Controls.DataGrid> controls now uses correct theme colors. For example:
 
   Before: 
 
-  ![Sort indicator arrow before accessibility improvements](./media/sort-indicator-before.png) 
+  ![Screenshot of sort indicator arrow before improvements.](./media/whats-new-in-accessibility/sort-indicator-before.png) 
```

Comment:
These images actually look wrong. I don't see much difference between them and I'm not seeing a sort indicator arrow.

@merriemcgaw am I missing something?
![image](https://user-images.githubusercontent.com/12971179/69317914-d57cbd80-0bf0-11ea-8cd7-38f86499b174.png)

Reply:
@mairaw no, I don't think you're missing anything. I didn't notice this when I was working on the article, but you are correct. I checked the published article and I'm seeing the same anomaly: [What's new in accessibility in .NET Framework 4.7.1](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility#whats-new-in-accessibility-in-net-framework-471).

Is there any action for me to take, or is this something your team will fix? Happy to help in any way I can.

Reply:
I'll file an issue for this.